components/CurrentProjects.jsx

```diff
@@ -0,0 +1,56 @@
+import getFormattedDate from '../util/getFormattedDate';
+import Show from './Show';
+
+export default function CurrentProjects({ currentProjects }) {
+  return (
+    <section className="cmp-current-projects">
+      <Show when={!currentProjects.length}>
+        <h2 className="cmp-current-projects__header cmp-current-projects__header--heading">Current Projects</h2>
```

Comment:
https://www.youtube.com/watch?v=ULdkpU51hTQ

Check this out! Happy to pair on this